zenstack generate fails with pnpm

**Description and expected behavior**
Generating with `zenstack generate` fails in Nuxt 3 project when using pnpm (works fine with npm).

**Environment (please complete the following information):**

-   ZenStack version: 1.0.0-beta.6
-   Prisma version: 4.16.2
-   Database type: MySQL

**Additional context**
I'm trying to setup ZenStack in a Nuxt project using tRPC but I'm stumbling on the following error when generating the initial client with pnpm:
```
⠋ Running plugin ZodError compiling generated code:
node_modules/.pnpm/@zenstackhq+runtime@1.0.0-beta.6_@prisma+client@4.16.2/node_modules/.zenstack/zod/input/UserInput.schema.ts:23:26 - error TS2742: The inferred type of 'UserInputSchema' cannot be named without a reference to '.pnpm/@prisma+client@4.16.2_prisma@4.16.2/node_modules/.prisma/client'. This is likely not portable. A type annotation is necessary.

23             export const UserInputSchema = {
                            ~~~~~~~~~~~~~~~
node_modules/.pnpm/@zenstackhq+runtime@1.0.0-beta.6_@prisma+client@4.16.2/node_modules/.zenstack/zod/input/UserInput.schema.ts:23:26 - error TS2742: The inferred type of 'UserInputSchema' cannot be named without a reference to '.pnpm/@prisma+client@4.16.2_prisma@4.16.2/node_modules/@prisma/client/runtime'. This is likely not portable. A type annotation is necessary.

23             export const UserInputSchema = {
                            ~~~~~~~~~~~~~~~
node_modules/.pnpm/@zenstackhq+runtime@1.0.0-beta.6_@prisma+client@4.16.2/node_modules/.zenstack/zod/objects/UserArgs.schema.ts:11:15 - error TS2742: The inferred type of 'UserArgsObjectSchema' cannot be named without a reference to '.pnpm/@prisma+client@4.16.2_prisma@4.16.2/node_modules/@prisma/client/runtime'. This is likely not portable. A type annotation is necessary.

11  export const UserArgsObjectSchema = Schema
                 ~~~~~~~~~~~~~~~~~~~~
node_modules/.pnpm/@zenstackhq+runtime@1.0.0-beta.6_@prisma+client@4.16.2/node_modules/.zenstack/zod/objects/UserSelect.schema.ts:11:15 - error TS2742: The inferred type of 'UserSelectObjectSchema' cannot be named without a reference to '.pnpm/@prisma+client@4.16.2_prisma@4.16.2/node_modules/@prisma/client/runtime'. This is likely not portable. A type annotation is necessary.

11  export const UserSelectObjectSchema = Schema
                 ~~~~~~~~~~~~~~~~~~~~~~
```

This is likely due to the way Nuxt handles `tsconfig.json`

Hello,

Same issue with a simpler project, and pnpm, but without Nuxt/any dependency, neither tsconfig.json, when trying `zenstack generate`, after `zenstack init`

```
⠋ Running plugin ZodError compiling generated code:
../node_modules/.pnpm/@zenstackhq+runtime@1.0.0-beta.3_@prisma+client@4.16.2/node_modules/.zenstack/zod/input/PostInput.schema.ts:22:26 - error TS2742: The inferred type of 'PostInputSchema' cannot be named without a reference to '.pnpm/@prisma+client@4.16.2_prisma@4.16.2/node_modules/.prisma/client'. This is likely not portable. A type annotation is necessary.

22             export const PostInputSchema = {
                            ~~~~~~~~~~~~~~~
../node_modules/.pnpm/@zenstackhq+runtime@1.0.0-beta.3_@prisma+client@4.16.2/node_modules/.zenstack/zod/input/PostInput.schema.ts:22:26 - error TS2742: The inferred type of 'PostInputSchema' cannot be named without a reference to '.pnpm/@prisma+client@4.16.2_prisma@4.16.2/node_modules/@prisma/client/runtime'. This is likely not portable. A type annotation is necessary.

(... more error like that)
```

--- a/packages/runtime/package.json
+++ b/packages/runtime/package.json
@@ -37,15 +37,13 @@
         "zod": "3.21.1",
         "zod-validation-error": "^0.2.1"
     },
-    "peerDependencies": {
-        "@prisma/client": "^4.0.0"
-    },
     "author": {
         "name": "ZenStack Team"
     },
     "homepage": "https://zenstack.dev",
     "license": "MIT",
     "devDependencies": {
+        "@prisma/client": "^4.0.0",
         "@types/bcryptjs": "^2.4.2",
         "@types/jest": "^29.5.0",
         "@types/lower-case-first": "^1.0.1",

--- a/packages/schema/package.json
+++ b/packages/schema/package.json
@@ -78,9 +78,6 @@
         "publish-dev": "pnpm publish --registry http://localhost:4873",
         "postinstall": "node bin/post-install.js"
     },
-    "peerDependencies": {
-        "prisma": "^4.0.0"
-    },
     "dependencies": {
         "@paralleldrive/cuid2": "^2.2.0",
         "@prisma/generator-helper": "4.10.0",

